coderd/database/migrations/000111_workspace_agent_metadata.up.sql

```diff
@@ -0,0 +1,15 @@
+-- TODO: Do we need an index for workspace_agent_id or is the multi-column PRIMARY
+-- key enough?
+CREATE TABLE workspace_agent_metadata (
+    workspace_agent_id uuid NOT NULL,
+    display_name text NOT NULL,
+    key character varying(128) NOT NULL,
+    script text NOT NULL,
+    value text NOT NULL DEFAULT '',
```

Comment:
I think we should have a limit in the database on size here. It'd be sucky if someone accidentally printed output in their script and filled up like 1GB lmao